src/workerd/server/workerd.c++

```diff
@@ -1150,6 +1246,11 @@ private:
     hadErrors = true;
   }
 
+#if _WIN32
+  kj::Promise<void> waitForChanges(FileWatcher& watcher) {
+    KJ_UNREACHABLE("Watching is not yet implemented on Windows");
```

Comment:
```suggestion
    KJ_UNIMPLEMENTED("Watching is not yet implemented on Windows");
```

`KJ_UNREACHABLE` does not take a parameter... does this compile? I guess `("...");` is being treated as a statement with no effect?